assets/airports/loww.json

```diff
@@ -386,448 +362,339 @@
+              "MASUR": ["_MASUR272009", ["MASUR", "A170-|S330"], ["TEMTA", "A60-|S250"], ["NERDU", "A70+|S250-"], ["WW987", "A70+|S250-"], ["WW985", "A70|S250-"], ["WW983", "A70|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "LANUX": ["_LANUX331017", ["LANUX", "A150-|S330"], ["NERDU", "A70+|S250-"], ["WW987", "A70+|S250-"], ["WW985", "A70|S250-"], ["WW983", "A70|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "GAMLI": ["LIMRA", ["GAMLI", "A60+|S330+"], ["BARUG", "A170-|S330"], ["WW450", "A90-|S250-"], ["BALAD", "A100|S250-"], ["WW988", "A80|S250-"], ["WW982", "A50-|S250-"], ["WW980", "A60-|S250-"], ["WW978", "A50-|S250-"], ["WW976", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "XANUT": ["_NIGSI202010", ["NIGSI", "A180-|S330"], ["BALAD", "A100|S250-"], ["WW988", "A80|S250-"], ["WW982", "A50-|S250-"], ["WW980", "A60-|S250-"], ["WW978", "A50-|S250-"], ["WW976", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "NATEX": ["_NATEX080019", ["NATEX", "A140-|S330-"], ["PESAT", "A60+|S250-"], ["WW991", "A70|S250-"], ["WW989", "A60+|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "MIKOV": ["_MIKOV345020", ["MIKOV", "A170-|S330"], ["MABOD", "A60+|S250-"], ["WW987", "A100|S250-"], ["WW985", "A80|S250-"], ["WW983", "A80|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]],
+              "REKLU": ["_REKLU082028", ["REKLU", "A180|S250-"], ["MABOD", "A60+|S250-"], ["WW987", "A100|S250-"], ["WW985", "A80|S250-"], ["WW983", "A80|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW673", "A50-|S250-"], ["WW672", "A50-|S250-"], ["WW671", "A50-|S250-"], ["WW670", "A50-|S250-"], ["_RWY16", "A50-|S250-"]]
+            },
+          "body": []
         }
     },
     "spawnPatterns": [
```

Comment:
Could you cite a source that backs up the assertion that they do not accept any flights outside of the 17 hour window?